AsyncDisplayKit/Details/_ASDisplayView.mm

```diff
@@ -202,6 +202,15 @@ - (void)willRemoveSubview:(UIView *)subview
 #endif
 }
 
+- (CGSize)sizeThatFits:(CGSize)size
+{
+  ASDisplayNode *node = _asyncdisplaykit_node; // Create strong reference to weak ivar.
+  if (node) {
+    return [node sizeThatFits:size];
+  }
+  return [super sizeThatFits:size];
```

Comment:
We should do if-else here, or ternary operator.

return (node ? [node sizeThatFits:size] : [super sizeThatFits:size]);